sklearn/model_selection/tests/test_split.py

```diff
@@ -368,24 +369,56 @@ def test_stratified_kfold_no_shuffle():
         list(StratifiedKFold(2).split(X, y1)),
         list(StratifiedKFold(2).split(X, y2)))
 
+    # Check equivalence to KFold
```

Comment:
why this test? this seems like a coincidence?

Reply:
@amueller requested it. Not a coincidence. A matter of ensuring dependencies are maintained